client/src/components/CommentBody/CommentBody.js

```diff
@@ -0,0 +1,31 @@
+
+import React from 'react';
+import "./CommentBody.css";
+
+// Function based React Component
+const CommentBody = (props) => {
+  // Default Class to apply to Component
+  // No props right now but when there is data we can use 
+  // this.props.comment and this.props.date
+  let classList = `CommentBody`;
```

Comment:
If you're not going to be updating/editing the class list, then just drop this into the JSX for the `className` property

Reply:
@cjohnston96 don't forget this one